src/group_theory/complement.lean

```diff
@@ -241,6 +241,13 @@ noncomputable def to_equiv (hS : S ∈ subgroup.left_transversals (H : set G)) :
   quotient.mk' (to_equiv hS q : G) = q :=
 (to_equiv hS).symm_apply_apply q
 
+@[to_additive] lemma range_to_equiv_apply {f : G ⧸ H → G} (hf : ∀ q, ↑(f q) = q) (q : G ⧸ H) :
+  ↑(to_equiv (range_mem_left_transversals hf) q) = f q :=
```

Comment:
It's almost always better to give an explicit coercion rather than using the little `↑` arrow. I'm not _sure_ about naming but I think just `to_equiv_apply` is better:
```suggestion
@[to_additive] lemma to_equiv_apply {f : G ⧸ H → G} (hf : ∀ q, (f q : G ⧸ H) = q) (q : G ⧸ H) :
  (to_equiv (range_mem_left_transversals hf) q : G) = f q :=
```